config/initializers/stripe.rb

```diff
@@ -1 +1,15 @@
 Stripe.api_key = ENV['STRIPE_API_KEY']
+
+StripeEvent.setup do
+  subscribe 'charge.succeeded' do |event|
+    user = User.where(customer_token: event.data.object.customer).first
+    Payment.create(user: user,
```

Comment:
Whenever you don't have a creation linked to an `if`/`else`, you should use `create!`. That way you will get a 500 raised and know a problem was experienced.